.golangci.yml

```diff
@@ -145,12 +152,14 @@ linters:
   enable:
```

Comment:
Normally, it is best to set golangci-lint either by disabling all linters and then only enabling the necessary ones. Or adding necessary ones on top the default. Maybe we can use that approach here.
Currently, the setup seems a bit conflicting.

Reply:
:+1: simplified it.